components/access_history_modal/access_history_modal.test.tsx

```diff
@@ -45,7 +43,7 @@ describe('components/AccessHistoryModal', () => {
             getUserAudits: jest.fn(),
         };
         const props = {...baseProps, actions};
-        const wrapper = shallow(
+        const wrapper: ShallowWrapper<any, any, AccessHistoryModal> = shallow(
```

Comment:
@devinbinnie @reflog I remember someone saying that we weren't going to change tests over to TypeScript because it would be stricter. Should we clarify in these tickets that we don't need tests to be changed over? I don't think there's anything wrong with keeping them with a PR once they're in though

Reply:
Whoops, I thought this was a Help Wanted PR. I guess we don't need to update anything 😛

Reply:
@hmhealey - you are correct, we didn't port tests to TS to save migration time. But in general - I am all for it :)